bikeshed/config/status.py

```diff
@@ -77,13 +78,14 @@
     "fido/PS": "Proposed Standard",
     "fido/FD": "Final Document"
 }
-snapshotStatuses = ["w3c/WD", "w3c/FPWD", "w3c/LCWD", "w3c/CR", "w3c/PR", "w3c/REC", "w3c/PER", "w3c/WG-NOTE", "w3c/IG-NOTE", "w3c/NOTE", "w3c/MO"]
-unlevelledStatuses = ["LS", "LD", "DREAM", "w3c/UD", "LS-COMMIT", "LS-BRANCH", "FINDING", "DRAFT-FINDING"]
+snapshotStatuses = ["w3c/WD", "w3c/FPWD", "w3c/LCWD", "w3c/CR", "w3c/PR", "w3c/REC", "w3c/PER", "w3c/WG-NOTE", "w3c/IG-NOTE", "w3c/NOTE", "w3c/MO", "whatwg/RD"]
+unlevelledStatuses = ["LS", "LD", "DREAM", "w3c/UD", "LS-COMMIT", "LS-BRANCH", "FINDING", "DRAFT-FINDING", "whatwg/RD"]
 deadlineStatuses = ["w3c/LCWD", "w3c/PR"]
-noEDStatuses = ["LS", "LS-COMMIT", "LS-BRANCH", "LD", "FINDING", "DRAFT-FINDING", "DREAM", "iso/NP"]
+noEDStatuses = ["LS", "LS-COMMIT", "LS-BRANCH", "LD", "FINDING", "DRAFT-FINDING", "DREAM", "iso/NP", "whatwg/RD"]
```

Comment:
I don't understand why LS is not prefixed, but this one is. And why is LS-COMMIT and LS-BRANCH not a snapshot status?

Reply:
LS (and related) aren't associated with any particular mega-group, but your Review Draft status is definitely whatwg-specific, so it's tagged accordingly.

Why they're not snapshot statuses is a good question, they probably should be.

Reply:
Snapshot statuses would require them to have a `Date` manually added, right? Whereas it's reasonable enough to have auto-generated dates for such snapshots, since they're only generated once and never updated.